pkg/api/grpcServer.go

```diff
@@ -42,6 +43,7 @@ import (
 type (
 	// grpcServer holds the dispatchState for the gRPC API server.
 	grpcServer struct {
+		dispatcher2      runtime.Dispatcher
```

Comment:
Can we just call it dispatcher? so don't need to go back to clean up later. The same for other dispatcher2.